internal/lm/nvml.go

```diff
@@ -254,3 +303,25 @@ func getDeviceClasses(devices []resource.Device) ([]uint32, error) {
 	}
 	return classes, nil
 }
+
+func generateUUIDs(seed string, count int) []string {
```

Comment:
Why does this need a count / loop? It will only ever generate one UUID, correct?

Reply:
done